src/Attribute/AttributeReader.php

```diff
@@ -77,4 +88,25 @@ class_exists($attribute->getName()) &&
             array_values($filteredAttributes),
         );
     }
+
+    /**
+     * @param array $variables
+     * @return list<AttributeInterface>
+     */
+    public function getEnvironmentAnnotations(array $variables): array
+    {
+        $labels = [];
+        /** @psalm-var mixed $value */
+        foreach ($variables as $variableName => $value) {
+            if (str_starts_with((string) $variableName, $this->envLabelPrefix)) {
+                $labelName = substr((string) $variableName, strlen($this->envLabelPrefix));
+                if ('' == $labelName) {
+                    continue;
+                }
+                $labels[] = new Label(strtolower($labelName), isset($value) ? (string) $value : null);
```

Comment:
We shouldn't convert label name to lowercase here because if we do so we are unable to specify camel-case labels like `parentSuite` (label names are case sensitive in allure).
This should be fixed in allure-js as well but that's another story.

Reply:
I have just copied the behavior from allure-js; don't like it either. I'm removing it.